transport/src/commonMain/kotlin/com/genesys/cloud/messenger/transport/core/MessagingClientImpl.kt

```diff
@@ -262,6 +275,12 @@ internal class MessagingClientImpl(
                     structuredMessage.events.forEach {
                         eventHandler.onEvent(it)
                     }
+                } else {
+                    structuredMessage.events.forEach {
+                        if (it.eventType == StructuredMessageEvent.Type.Presence) {
```

Comment:
This triggers `onEvent` calls for customer joins, right? (just making sure I understand)

Reply:
correct. Want to highlight that it is true for Inbound messages only, meaning that it will trigger the Event.ConversationAutostart ONLY when the Presence (autostart) was sent by customer.